src/types/index.ts

```diff
@@ -11,12 +11,14 @@ import {
 import { IpcSender } from 'lib/ipc/ipcService';
 import * as PLN from 'lib/lightning/types';
 import { PolarPlatform } from 'utils/system';
+import { AutoMineMode } from 'store/models/network';
```

Comment:
I think it would make sense to move the declaration of the `AutoMineMode` enum into this file instead of importing from the store here.